website/docs/docs/build/unit-tests.md

```diff
@@ -0,0 +1,253 @@
+---
+title: "Unit tests"
+sidebar_label: "Unit tests"
+description: "Read this tutorial to learn how to use unit tests on your SQL models."
+search_weight: "heavy"
+id: "unit-tests"
+keywords:
+  - unit test, unit tests, unit testing, dag
+---
+:::note closed beta
+
+Unit testing is currently in closed beta for dbt Cloud accounts that have updated to a [versionless environment](/docs/dbt-versions/upgrade-core-in-cloud).
+
+It is available now as an alpha feature for dbt Core v1.8 users.
+
+:::
+
+Historically, the test coverage capabilities of dbt were limited to “data” tests &dash; testing the quality of input data or the shape of the resulting datasets &mdash; that could only be executed *after* a model had been built. 
```

Comment:
should we link to the docs for data tests?

Reply:
```suggestion
Historically, the test coverage capabilities of dbt were limited to [“data” tests](/docs/build/data-tests) &dash; testing the quality of input data or the shape of the resulting datasets &mdash; that could only be executed *after* a model had been built. 
```